uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+2. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
+3. Select a test and run it.
+    - Note: Once the test starts, avoid interacting with your machine (e.g. move mouse, use keyboard) to allow the test to navigate the Dev Home app and execute the test.
```

Comment:
Is it possible to run this in a VM? Assuming everyone uses the UITest project in the future, we'll have possibly hours of tests. ;)